setup_environment.sh

```diff
@@ -0,0 +1,100 @@
+
+# download, install and configure java8 jdk
+wget --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/8u102-b14/jdk-8u102-linux-x64.tar.gz
+echo "Unpacking JDK8..."
+tar -xzf jdk-8*
+sudo mv ./jdk1.8* /usr/lib/jvm/jdk1.8.0_102
+sudo update-alternatives --install "/usr/bin/java" "java" "/usr/lib/jvm/jdk1.8.0_102/bin/java" 1
+sudo update-alternatives --install "/usr/bin/javac" "javac" "/usr/lib/jvm/jdk1.8.0_102/bin/javac" 1
+sudo update-alternatives --install "/usr/bin/javaws" "javaws" "/usr/lib/jvm/jdk1.8.0_102/bin/javaws" 1
+sudo chmod a+x /usr/bin/java
+sudo chmod a+x /usr/bin/javac
+sudo chmod a+x /usr/bin/javaws
+
+printf "${RED}[IMPORTANT]${NOCOLOR} Pick JDK8 option!\n"
+sudo update-alternatives --config java
+printf "${RED}[IMPORTANT]${NOCOLOR} Pick JDK8 option!\n"
+sudo update-alternatives --config javac
+printf "${RED}[IMPORTANT]${NOCOLOR} Pick JDK8 option!\n"
+sudo update-alternatives --config javaws
+
+# export jdk environmental variables
+export PATH=$PATH:/usr/lib/jvm/jdk1.8.0_102/bin
+export JAVA_HOME=/usr/lib/jvm/jdk1.8.0_102
+export JAVA7_HOME=/usr/lib/jvm/jdk1.7.0_79
+export JAVA8_HOME=/usr/lib/jvm/jdk1.8.0_102
+
+# download, install and configure android sdk
+mkdir -p $HOME/android-sdk/
+wget https://dl.google.com/android/android-sdk_r24.4.1-linux.tgz
+echo "Unpacking Android SDK..."
+tar -xzf android-sdk_r24.4.1-linux.tgz
+cd android-sdk-linux/
+mv * $HOME/android-sdk/
+
+# export sdk environmental variables
+export ANDROID_HOME=$HOME/android-sdk/
+export PATH=$PATH:$HOME/android-sdk/tools
+
+# set needed environmental variables permanent for current user
+touch $HOME/android-sdk/setup-activityteam-development.sh
+chmod a+rwx $HOME/android-sdk/setup-activityteam-development.sh
+echo '#!/bin/bash' >> $HOME/android-sdk/setup-activityteam-development.sh
+echo "export PATH=\$PATH:/usr/lib/jvm/jdk1.8.0_102/bin" >> $HOME/android-sdk/setup-activityteam-development.sh
+echo "export JAVA_HOME=/usr/lib/jvm/jdk1.8.0_102" >> $HOME/android-sdk/setup-activityteam-development.sh
+echo "export JAVA7_HOME=/usr/lib/jvm/jdk1.7.0_79" >> $HOME/android-sdk/setup-activityteam-development.sh
+echo "export JAVA8_HOME=/usr/lib/jvm/jdk1.8.0_102" >> $HOME/android-sdk/setup-activityteam-development.sh
+echo "export ANDROID_HOME=${HOME}/android-sdk/" >> $HOME/android-sdk/setup-activityteam-development.sh
+echo "export PATH=\$PATH:${HOME}/android-sdk/tools" >> $HOME/android-sdk/setup-activityteam-development.sh
+echo "source ${HOME}/android-sdk/setup-activityteam-development.sh" >> $HOME/.bashrc
+
+# download and update essential sdk files
+android update sdk --no-ui
+
+# libraries needed by aapt and adb
+sudo apt-get install libstdc++6:i386 libgcc1:i386 zlib1g:i386 libncurses5:i386
+
+# kill current user session in order to run setup-activityteam-development.sh
+printf "${GREEN}[OK]${NOCOLOR} Log out in 3 seconds.\n"
+sleep 3s
+pkill -u $USER
```

Comment:
Is it really necessary to kill user session ? 
In my opinion more gentle approach is not to hardcode any PATH modification in .bashrc and do it "on demand".
By "on demand" I mean asking user to run `source $HOME/android-sdk/setup-activityteam-development.sh` every time before starting project development in given terminal.

In such case user could use his machine to develop other projects, requiring different enviroment.